docs/pages/logging-and-metrics/client-monitoring.md

```diff
@@ -4,7 +4,7 @@ Lodestar has the ability to send client stats to a remote service for collection
 At the moment, the main service offering remote monitoring is [beaconcha.in](https://beaconcha.in/).
 
 Instructions for setting up client monitoring with _beaconcha.in_ can be found in their docs about
-[Mobile App <> Node Monitoring](https://kb.beaconcha.in/beaconcha.in-explorer/mobile-app-less-than-greater-than-beacon-node)
+[Mobile App &lt&gt; Node Monitoring](https://kb.beaconcha.in/beaconcha.in-explorer/mobile-app-less-than-greater-than-beacon-node)
```

Comment:
ooof... this stinks.  Is it for react bullhonkey so it doesn't think its an empty tag?  I noticed the sanitizer and guessing so but figured it was worth the question.

Reply:
This is html entities, apparently markdown is not supposed to support special characters. See https://daringfireball.net/projects/markdown/syntax#autoescape